src/bci_build/package/appcontainers.py

```diff
@@ -434,7 +435,11 @@ def _get_nginx_kwargs(os_version: OsVersion):
     ApplicationStackContainer(
         name="helm",
         pretty_name="Kubernetes Package Manager",
-        from_image=f"{_build_tag_prefix(os_version)}/bci-micro:{OsContainer.version_to_container_os_version(os_version)}",
+        from_image=(
```

Comment:
nit: can't we simplify this? The only bit that changes is the prefix.

Reply:
moved into a helper function, since we need it many times.